bve-corpus/src/bin/bve-parser-run.rs

```diff
@@ -13,7 +14,8 @@ arg_enum! {
         B3D,
         CSV,
         Animated,
-        TrainDat
+        TrainDat,
+        ExtensionsCfg,
```

Comment:
yay trailing comma so this line won't be marked as changed next pr

Reply:
I blame the fact it's in a macro so rustfmt doesn't touch it :)